.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   root: true,
-  extends: "babel",
+  parser: "lightscript-eslint",
```

Comment:
I see, the extends refers to `eslint-config-babel`, which we haven't built yet... I imagine we'll want one though...